src/core/tsi/s2a/record_protocol/s2a_crypter.h

```diff
@@ -213,4 +257,58 @@ grpc_status_code s2a_write_tls13_record(
     size_t unprotected_vec_size, iovec protected_record, size_t* bytes_written,
     char** error_details);
 
+/** This method populates |plaintext_size| with the maximum size (in bytes) of
+ *  the plaintext obtained by decrypting |protected_vec| using |crypter|. This
+ *  method assumes that |protected_vec| contains a TLS payload of type
+ *  "application data"; in particular, |protected_vec| is assumed to not contain
+ *  the record header.
+ *  - crypter: an instance of s2a_crypter, which must have been initialized
+ *    using the s2a_crypter_create method.
+ *  - protected_vec: a pointer to a buffer of iovec's that contain the payload
+ *    of a TLS 1.3 record.
+ *  - protected_vec_size: the size of the |protected_vec| buffer.
```

Comment:
Don't think it's a good idea to take these two arguments when you only need the size in bytes. Seems cleaner to let the caller handle this (also see above about not taking the size at all).

Reply:
I've now removed this method, and the caller will use the normal `s2a_max_plaintext_size` method to determine how much memory they must allocate for a decrypt operation.